vllm/config.py

```diff
@@ -113,6 +113,12 @@ def _verify_quantization(self) -> None:
                 f"{supported_quantization}.")
         self.quantization = quantization
 
+    @staticmethod
+    def hf_config_get_num_layers(hf_config):
+        if getattr(hf_config, "model_type", None) == "chatglm":
+            return hf_config.num_layers
+        return hf_config.num_hidden_layers
```

Comment:
Please remove this function and add the following to `ChatGLMConfig`

```python
    attribute_map = {
        "num_hidden_layers": "num_layers",
    }
```

You can refer to falcon's config: https://github.com/vllm-project/vllm/blob/8516999495114926c9838c2d6e0feb580d4d983f/vllm/transformers_utils/configs/falcon.py#L25-L29